### ISSUE TITLE
Link no longer working https://chefspec.github.io/chefspec/

### ISSUE THREAD

**Issue by wswheeler** [chefspec/chefspec#951]
It appears that the link provided at the top of the github repo no longer works. It was great content.

**Comment by iennae** [comment]
Yeah, it's super frustrating to want to point folks to this project and it's missing that helpful information. If it's out of date, please update the header and remove the link to it from the project.
